### ISSUE TITLE
About me popup

### ISSUE THREAD

**Issue by obed-smart** [steadydogz/profile-card#2]
HTML:the document is not accessible enough to assistive technology like screen readers and also the keyboard navigation is not user friendly so the html need to be updated 

STYLE:the styling does not much issue but the problem there is that the focus mood is not smooth and well styled so work on the focus and active state.

JAVASCRIPT: I noticed that when you're using keyboard navigation, it's supposed to return to the top of the about me page but it's leave the entire page and this along makes it not user friendly.

**Comment by steadydogz** [comment]
NA YOU BUILD THE FEATURE NA, DEAL WITH IT !

**Comment by obed-smart** [comment]
> NA YOU BUILD THE FEATURE NA, DEAL WITH IT !

> I no call your name OGA

**Comment by obed-smart** [comment]
> > NA YOU BUILD THE FEATURE NA, DEAL WITH IT !

**Comment by obed-smart** [comment]
@steadydogz i no call your name so rest

**Comment by obed-smart** [comment]
it has been fixed
